backend/src/main/java/gov/cdc/usds/simplereport/service/OktaServiceImpl.java

```diff
@@ -144,7 +144,7 @@ private String getOrganizationExternalIdFromGroupName(String groupName, Organiza
     }
 
     private String generateGroupDescription(String orgName, OrganizationRole role) {
-        return String.format("%s - %s", orgName, role.getDescription());
+        return String.format("%s - %ss", orgName, role.getDescription());
```

Comment:
Woe unto us should we add an "ox" role :P

Reply:
As long as we call it "Ox user" we'll be fine. 🤣